qcore/im.py

```diff
@@ -178,5 +245,17 @@ def get_period_unit(self):
 
     @staticmethod
     def from_im_name(name: str):
+        """Create an IM from a string.
+
+        Parameters
+        ----------
+        name : str
+            The name of the intensity measure (e.g. "PGV")
```

Comment:
Maybe a better example could be pSA_1.0, just so people get the idea that it also can include the period not just the IM type